doc/subport/sonic-sub-port-intf-hld.md

```diff
@@ -96,63 +102,98 @@ A sub port interface shall support the following features:
 * VRF
 * RIF counters
 * QoS setting inherited from parent physical port or port channel
-* mtu inherited from parent physical port or port channel
+* MTU: 
+  MTU inherited from parent physical port or port channel. 
+  If subinterface MTU is configured, MTU on subinterface will be configured with:
+  - If Subinterface MTU <= parent port MTU, configured subinterface MTU will be applied.
+  - If Subinterface MTU > parent port MTU, parent port MTU will be applied.
```

Comment:
Dont we throw an error in this case?

Reply:
As discussed in review meeting, user can change MTU on parent interface. Restricting this update depending on MTU of subinterfaces will be hard for user since there could be 100s of subinterfaces under the parent interface.